src/zzz_od/application/driver_disc_read/driver_disc_read_app.py

```
@@ -0,0 +1,625 @@
+                    global_index += 1
+
+        finally:
+            # 停止 worker 线程
+            self._stop_workers()
+            self._wait_collector_thread()
+
+        log.info(f'扫描完成 共识别:{len(self.disc_data_dict)}个')
+        return self.round_success(f'识别完成: {len(self.disc_data_dict)}个')
+
+    def _start_workers(self):
+        """启动 worker 进程（GPU 多进程）"""
```

Ruff (0.14.0)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)